usagov/vote-gov-drupal
# VOTE-2272 replace touchpoints global button modal with a link to a hosted form

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-2272

## Description

Replace global touchpoint button modal behavior with a link to a hosted form

## Deployment and testing

### Post-deploy steps

1. `lando drush cr`

### QA/Testing instructions

1. visit http://vote-gov.lndo.site/ and confirm button opens a touchpoints form in a new window
2. visit http://vote-gov.lndo.site/es and confirm feedback button is removed

## Checklist for the Developer




## Checklist for the Peer Reviewers
